### ISSUE TITLE
Run transfer learning training + testing 

### ISSUE THREAD

**Issue by vcheplygina** [vcheplygina/cats-scans#37]
Replicate one of Irma's results by running the code again on the server

**Comment by FSchijve** [comment]
![Irma's results replicated](https://user-images.githubusercontent.com/58231888/101892428-a9d5c800-3ba3-11eb-9ee0-e2e76f09f2d5.png)

Replicated Irma's result of pretraining ResNet on ImageNet and having target set ISIC.
As shown, my obtained average AUC score is 0.9408, Irma's result had an AUC score of 0.946.

Settings that I copied:  
Image size: 112 x 112
Batch size: 128
Learning rate: 1.0e-05 
No learning rate rescheduler
Epochs: 70
